test/test_scanmap.py

```diff
@@ -0,0 +1,112 @@
+        self.assertEqual(b'\x97', get_scan_codes('T'))
+        self.assertEqual(b'\x98', get_scan_codes('U'))
+        self.assertEqual(b'\x99', get_scan_codes('V'))
+        self.assertEqual(b'\x9a', get_scan_codes('W'))
+        self.assertEqual(b'\x9b', get_scan_codes('X'))
+        self.assertEqual(b'\x9c', get_scan_codes('Y'))
+        self.assertEqual(b'\x9d', get_scan_codes('Z'))
+
+        self.assertEqual(b'\x27', get_scan_codes('0'))
+        self.assertEqual(b'\x1e', get_scan_codes('1'))
+        self.assertEqual(b'\x1f', get_scan_codes('2'))
+        self.assertEqual(b'\x20', get_scan_codes('3'))
+        self.assertEqual(b'\x21', get_scan_codes('4'))
+        self.assertEqual(b'\x22', get_scan_codes('5'))
+        self.assertEqual(b'\x23', get_scan_codes('6'))
+        self.assertEqual(b'\x24', get_scan_codes('7'))
+        self.assertEqual(b'\x25', get_scan_codes('8'))
+        self.assertEqual(b'\x26', get_scan_codes('9'))
+
+        self.assertEqual(b'\x2b', get_scan_codes('\t'))
+        self.assertEqual(b'\x28', get_scan_codes('\n'))
+
+        self.assertEqual(b'\x9e', get_scan_codes('!'))
+        self.assertEqual(b'\xb4', get_scan_codes('"'))
+        self.assertEqual(b'\xa0', get_scan_codes('#'))
+        self.assertEqual(b'\xa1', get_scan_codes('$'))
+        self.assertEqual(b'\xa2', get_scan_codes('%'))
+        self.assertEqual(b'\xa4', get_scan_codes('&'))
+        self.assertEqual(b'\x34', get_scan_codes("'"))
+        self.assertEqual(b'\xa6', get_scan_codes('('))
+        self.assertEqual(b'\xa7', get_scan_codes(')'))
+        self.assertEqual(b'\xa5', get_scan_codes('*'))
+        self.assertEqual(b'\xae', get_scan_codes('+'))
+        self.assertEqual(b'\x36', get_scan_codes(','))
+        self.assertEqual(b'\x2d', get_scan_codes('-'))
+        self.assertEqual(b'\x37', get_scan_codes('.'))
+        self.assertEqual(b'\x38', get_scan_codes('/'))
+        self.assertEqual(b'\xb3', get_scan_codes(':'))
+        self.assertEqual(b'\x33', get_scan_codes(';'))
+        self.assertEqual(b'\xb6', get_scan_codes('<'))
+        self.assertEqual(b'\x2e', get_scan_codes('='))
+        self.assertEqual(b'\xb7', get_scan_codes('>'))
+        self.assertEqual(b'\xb8', get_scan_codes('?'))
+        self.assertEqual(b'\x9f', get_scan_codes('@'))
+        self.assertEqual(b'\x2f', get_scan_codes('['))
+        self.assertEqual(b'\x32', get_scan_codes('\\'))
+        self.assertEqual(b'\x30', get_scan_codes(']'))
+        self.assertEqual(b'\xa3', get_scan_codes('^'))
+        self.assertEqual(b'\xad', get_scan_codes('_'))
+        self.assertEqual(b'\xaf', get_scan_codes('{'))
+        self.assertEqual(b'\xb0', get_scan_codes('}'))
+        self.assertEqual(b'\xb2', get_scan_codes('|'))
+        self.assertEqual(b'\xb5', get_scan_codes('~'))
+
+        self.assertEqual(b'\x04\x05\x06', get_scan_codes('abc'))
+        with self.assertRaises(ValueError):
+            get_scan_codes('ööööö')
```

Comment:
Is the number of `ö` important? Why five?

Reply:
Good point, updated.